### ISSUE TITLE
dmg file not working on Mac

### ISSUE THREAD

**Issue by mmuppa** [chiragsakhuja/lc3v2#39]
The latest release does not work on a Mac (Intel chip). A blank screen shows up. 
<img width="1344" alt="lc3" src="https://github.com/chiragsakhuja/lc3tools/assets/11671387/c9a94095-4324-4763-b56e-1882b23fc34a">

**Comment by demirakbal** [comment]
Did you fix this issue somehow? I get the same exact error. I tried building the file from the source code but it still doesn't work.

**Comment by mmuppa** [comment]
No, I did not fix it. I found an online simulator instead: https://lc3.cs.umanitoba.ca/
[https://github.com/aidan-garvey/WebLC3/blob/main/static/weblc3_preview.png]<https://lc3.cs.umanitoba.ca/>
WebLC3 Simulator<https://lc3.cs.umanitoba.ca/>
Web-based editor and simulator for Little Computer 3 (LC-3)
lc3.cs.umanitoba.ca


Best,
Menaka



Menaka Abraham, M.S.
Teaching Professor
Computer Science & Systems
School of Engineering & Technology

***@***.******@***.***>

Office Location: CP130

Zoom url: https://tinyurl.com/menakaofficehours
________________________________
